Automattic/newspack-plugin
# Feat/update all transactional email templates

### All Submissions:






### Changes proposed in this Pull Request:



This PR updates the remaining transactional email template designs to match the updates done to the OTP email (https://github.com/Automattic/newspack-plugin/pull/3032, https://github.com/Automattic/newspack-plugin/pull/3033, and https://github.com/Automattic/newspack-plugin/pull/3034)

Account verification:
![Screenshot 2024-04-22 at 17 22 27](https://github.com/Automattic/newspack-plugin/assets/17905991/40922398-40dc-4a73-b609-23dfeb1b525c)

Reset password:
![Screenshot 2024-04-22 at 17 22 04](https://github.com/Automattic/newspack-plugin/assets/17905991/6967de76-c9ec-4013-a475-26590184de2c)

Receipt:
![Screenshot 2024-04-22 at 17 21 57](https://github.com/Automattic/newspack-plugin/assets/17905991/fd4a760f-48a6-4ce3-8307-6cb0df071a8f)

Account deletion:
![Screenshot 2024-04-22 at 17 22 10](https://github.com/Automattic/newspack-plugin/assets/17905991/1b86fff8-eb85-4a51-bffb-2de1d102ce29)

Subscription/Donation cancellation:
![Screenshot 2024-04-22 at 17 21 46](https://github.com/Automattic/newspack-plugin/assets/17905991/c389efe9-4b35-48b7-ae04-99393f26f2c2)

(OTP will have already been updated at this point)

Note: I believe we will be removing the Magic Link template in favor of always using the OTP. At least based on the Figma here: https://www.figma.com/file/c6T1goPQpkchdh7XBAM6aH/Reader-Activation-System?type=design&node-id=1589-35127&mode=design&t=1gF6i52UIhsMYAk3-0

I will confirm this and remove in a follow-up PR if this is the case. Regardless, I've updated the design for this template in this PR anyway, but currently there is no way to trigger it via the UI.

### How to test the changes in this Pull Request:

1. Go to Newspack > Engagement > Show Advanced Settings > Transactional Email Content, then trash each of the emails shown here via the edit button (should be every one mentioned above except for the receipt template). This is so our default templates can be used for emails.
2. Go to Newspack > Reader Revenue > Emails, then trash the receipt email via the edit button.
3. Trigger each of the emails, then view them and verify they look as pictured above. You can trigger each email as follows:
  - Account verification - As a logged out user, sign up for a new account. Once logged in, go to My Account and select the `Send me a link` option to send a verification email
  - Password reset - Once verified, go back to My Account and select the Create a Password link
  - Receipt - Purchase a donation via the donation block. You will need to set up the block on any post or page to do this
  - Account deletion - Go to My Account and select the Delete Account button
  - Receipt - Purchase a recurring donation or subscription product
  - Subscription/Donation Cancellation - Cancel the recurring donation or subscription via My Account

### Other information:
